src/frontend/src/pages/index.js

```diff
@@ -17,7 +17,7 @@ export default function IndexPage() {
           }
         }
       `}
-      render={(data) => <Layout telescopeUrl={data.site.siteMetadata.telescopeUrl} />}
```

Comment:
If you remove this, we don't need the StaticQuery or graphql sutff above this either.  It was only there due to the class component.

Reply:
I think we should consider pulling all the code in `Layout.jsx` into this `pages/index.js` file, since what you're really doing here is rendering that content.

Reply:
Fixed